clean.cmd

```diff
@@ -0,0 +1,177 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
I think we should think about this a little more I could be open to a "t" or "s" option as well but I'm not sure it is worth the complexity.